lib/client.js

```diff
@@ -20,6 +20,60 @@ if (!STATUS_CODES[508]) STATUS_CODES[508] = 'Loop Detected';
 const DEFAULT_SLOW_TIME = api.common.duration('2s');
 
 // HTTP Client interface for Impress Application Server
+// Properties:
+//   startTime <Date> client's start time
+//   req <http.IncomingMessage>
+//   res <http.ServerResponse>
+//   websocket <Object> | <null> initialized
+//       while processing a websocket request
+//   socket <net.Socket> socket associated with the connection
+//   server <Object> | <null> server instance
+//   application <Application> instance of impress.Application
+//   dynamicHandler <boolean> flag, true if client has been dispatched,
+//       default: false
+//   query <Object> collection of key and value pairs from url query
+//   schema <string> if server's transport is `tls`,
+//       is set to `https`, otherwise - `http`
+//   method <string> the request method in lower case
+//   access <Object> client access options
+//   fields <Object> parsed client data according to request content-type,
+//   files <Object> file uploads object, where the property names
+//       are field names and the values are arrays of file objects
+//   slowTime <number> client slow time in milliseconds
+//   timedOut <boolean> flag, is set to `true` if client hasn't been finished
+//       before server timeout, default: `false`
+//   finished <boolean> end response flag, default: `false`
+//   url <string> the path portion of the request url
+//   host <string> request host without port
+//   path <string> the path portion of the request url
+//       with trailing slash at the end if there isn't one
+//   pathDir <string> absolute path to path in request url
+//       in application's `www` directory
+//       (e.g. `/ias/applications/example/www/path/to/dir`)
+//   realPath <string> path to the nearest existent folder
+//   realPathDir <string> absolute path to the nearest existent folder
+//       in application's `www` directory
+//   execPath <string> path to the nearest directory
+//   execPathDir <string> absolute path to the nearest directory
+//       with handler script file for the request
+//       in application's `www` directory
+//   ext <string> extension of file in the path portion of the request url
+//   typeExt <string> extension type of returned data
+//   data <string> data received from client
+//   context <Object> client context
+//   chunks <Array> used to handle large requests receiving in chunks
```

Comment:
IMO this looks more like internal property.

Reply:
@SemenchenkoVitaliy so this shouldn't be documented, should it?